src/backend/gl/src/command.rs

```diff
@@ -222,18 +203,21 @@ impl RawCommandBuffer {
     }
 
     fn push_cmd(&mut self, cmd: Command) {
-        if self.take_access() {
-            let mut cmd_buffer = unsafe { &mut (*self.memory.get()).commands };
+        let slice = {
+            let mut memory = self.memory.lock().unwrap();
```

Comment:
So pushing command while the pool is in access is now legal ? You could also use `try_lock` and keep the restriction.

Reply:
sounds like a good idea